sygnal/apnspushkin.py

```diff
@@ -299,6 +299,13 @@ def _get_payload_event_id_only(self, n):
         if n.counts.missed_calls is not None:
             payload["missed_calls"] = n.counts.missed_calls
 
+        payload["aps"] = {"mutable-content": 1}
+        if (
+            n.devices[0].data["fallback_content"] is not None
+            and n.devices[0].data["fallback_content"]
```

Comment:
I'm not sure about that, the `fallback_content` will be an additional field in `data` property of https://matrix.org/docs/spec/client_server/r0.6.1#post-matrix-client-r0-pushers-set

Reply:
The thing is, for client implementations to know what to set, and for pushgateway implementations to know how to interpret it, it has to be specced, otherwise this becomes secret knowledge which will only work with riot-ios and sygnal.

Reply:
as per https://github.com/matrix-org/matrix-doc/pull/2631#issuecomment-646535932: no, it doesn't need to be in the spec. Sorry!

That said, I *do* think it would be useful if we documented somewhere within the sygnal project what parameters the `apnspushkin` supports in its `data` dict and what they do. Perhaps you could add these details under https://github.com/matrix-org/sygnal#app-types ?